docs/compilers/CSharp/Runtime Async Design.md

```diff
@@ -8,28 +8,14 @@ compiled with runtime async or not.
 
 ## Supporting runtime apis
 
-We use the following runtime flag to drive whether feature can be used. This flag must be defined in the same assembly that defines `object`, and the assembly cannot reference any other assemblies. In terms of
-CoreFX, this means it must be defined in the `System.Runtime` reference assembly.
-
-TODO: Determine whether just the presence of this flag will cause the compiler to generate in runtime async mode.
-
-```cs
-namespace System.Runtime.CompilerServices;
-
-public static class RuntimeFeature
-{
-    public const string Async = nameof(Async);
```

Comment:
FYI @agocke @VSadov @jakobbotsch The [runtime speclet](https://github.com/dotnet/runtime/blob/main/docs/design/specs/runtime-async.md#i845-sync-and-async-methods) still has mention of this flag, could it be removed?

"Async method definitions are only valid inside async-capable assemblies. An async-capable assembly is one which references a corlib containing an abstract sealed class RuntimeFeature with a public const string field member named Async."